packages/dependency-extraction-webpack-plugin/lib/util.js

```diff
@@ -73,6 +77,14 @@ function defaultRequestToExternalModule( request ) {
 		// which forces @wordpress/interactivity imports to be hoisted to static imports.
 		return `module ${ request }`;
 	}
+
+	const isWordPressScript = Boolean( defaultRequestToExternal( request ) );
+
+	if ( isWordPressScript ) {
+		return new Error(
+			`Attempted to use WordPress script in a module: ${ request }`
```

Comment:
Maybe add _not supported yet_ to the error to indicate that this will be eventually supported?

```suggestion
			`Attempted to use WordPress script in a module: ${ request }, which is not supported yet.`
```